source/dataman/export/AMExporter.cpp

```diff
@@ -53,6 +53,14 @@ const QMetaObject* AMExporter::getMetaObject(){
 	return metaObject();
 }
 
+QString AMExporter::currentRunExportFilePath()
+{
+	if (currentScan_)
+		return krRunName();
```

Comment:
It might be "Keyword Replacement". Not 100% sure though. There is a comment in the header file for all the `krXXX()` functions:
```
	///////////////////////////////
	// functions to implement the keyword replacement system
	///////////////////////////////
```